### DISCUSSION TITLE
Recurring tasks

### DISCUSSION THREAD

**Discussion by DoraDora74** [davish/obsidian-full-calendar#402] [Feature Requests]
I'm trying add an event and set it as recurring and as a task, but when I click to save, the event is not a task. 

Single events can be tasks no problem.

Not sure this is relevant but I'm on Linux.

**Comment by davish** [comment]
Recurring tasks have some strange edge cases that are not yet implemented. Thanks for submitting, I will change this in to a feature request.

**Comment by DoraDora74** [comment]
Thank you.

**Comment by D-L13W** [comment]
+1 to have this feature. To be specific, having the ability to recur tasks and then check them individually would be good (basically like in google calendar).

**Comment by YouFoundJK** [comment]
This works in my fork [here](https://github.com/YouFoundJK/plugin-full-calendar/releases/latest), if you are still interested.
